Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM ubuntu:16.04
-ARG sssd=false
```

Comment:
this is... important for the boulder use case?

Reply:
I believe that isn't used anywhere else in the Dockerfile which is why i removed it.

Reply:
restored ARG.